go.mod

```diff
@@ -5,5 +5,5 @@ go 1.14
 require (
```

Comment:
can we also bump the go version? (line 3)

Reply:
Solved here https://github.com/in4it/aws-config-compliance-checks/pull/7/commits/5029c9409b220fa10b85c83ec15ece42fe269498